### ISSUE TITLE
Restarting dense_stereo from a given image?

### ISSUE THREAD

**Issue by JakeC2i** [colmap/colmap#164]
I started the dense_stereo from GUI on 170 images yesterday. After over 20 hours I had to stop and as a result I got all photometric normal and depth maps, and only about 35% percent of geometric ones. I have two (groups of) questions:

1. What will happen if I restart dense_stereo now? Will it start from were I left off, or will it overwrite maps beginning from the first image? Is it possible to force the process to start from a given image? I didn't find anything about it FAQ nor anywhere else.
2. Is what I have enough to successfully fuse maps into the model and then to mesh it? I'm able to start the fusion, but it takes so ridiculous amount of time that I'd rather know in advance.

Thanks you.

**Comment by ahojnnes** [comment]
1. You can simply start dense_stereo again and it will start from where it left. The old results will be skipped and not overwritten. 
2. You can start the fusion on partial depth and normal map input. So, if you cancel dense_stereo before it finishes, you can still start dense_fusion. It will output a few warnings which you can ignore. Of course, the results won't be as complete and accurate as with the full input. 

You can also speed up the dense reconstruction significantly. Please refer to the FAQ section in the documentation for details. 

Let me know if you have other questions.

**Comment by JakeC2i** [comment]
@ahojnnes Thank you for your answer. Indeed I've managed to reconstruct the model with partial depth and normal data, but I'm hoping to get better quality after finishing dense_stereo, which I've just started.

I'll try to speed up dense reconstruction in future projects, for this one I left the configuration unchanged so the quality of maps would be consistent. Also, I'll first try different configurations on a smaller dataset ;)
